README.rst

```diff
@@ -580,6 +582,11 @@ candidate list further down.
     the `generic` type and eventually contributed back to this specification
   - as for other `type`, the `name` component is mandatory. In the worst case
     it can be a file or directory name.
+  - A `homepage_url` may be provided in `qualifiers`; this identifies the
```

Comment:
LGTM! I can merge alright but without the qualifiers part if that's OK with you.

Reply:
I don't understand what you mean by "without the qualifiers part". It needs to go somewhere, and presumably that would be in the qualifiers. If you just think it's clear from context, I guess that's fine as long as it's clear. We all want a clear spec :-).

Reply:
Sorry if this came out as not really clear: by "without the qualifiers part" I meant without the part about qualifiers above https://github.com/package-url/purl-spec/pull/92/files#r504010637 that would be best in a separate PR.

Reply:
Oh I see! Sure, that'd be great. Improvements are improvements.